examples/quickstart/10-configmaps.jsonnet

```diff
@@ -0,0 +1,233 @@
+# Copyright 2017 Heptio Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+local k = import "ksonnet.beta.2/k.libsonnet";
+local configMap = k.core.v1.configMap;
+
+local conf = {
+    namespace: "heptio-sonobuoy",
+    sonobuoyCfg: {
+        name: "sonobuoy-config-cm",
+    },
+    pluginsCfg: {
+        name: "sonobuoy-plugins-cm",
+    },
+    labels: {
+        component: "sonobuoy",
+    },
+};
+
+local sonobuoyConfigData = {
+    Description: "EXAMPLE",
+    Version: "v0.8.0",
+    ResultsDir: "/tmp/sonobuoy",
+    Resources: [
+        "CertificateSigningRequests",
+        "ClusterRoleBindings",
+        "ClusterRoles",
+        "ComponentStatuses",
+        "Nodes",
+        "PersistentVolumes",
+        "PodSecurityPolicies",
+        "ServerVersion",
+        "StorageClasses",
+        "ThirdPartyResources",
+        "ConfigMaps",
+        "DaemonSets",
+        "Deployments",
+        "Endpoints",
+        "Events",
+        "HorizontalPodAutoscalers",
+        "Ingresses",
+        "Jobs",
+        "LimitRanges",
+        "PersistentVolumeClaims",
+        "Pods",
+        "PodLogs",
+        "PodDisruptionBudgets",
+        "PodPresets",
+        "PodTemplates",
+        "ReplicaSets",
+        "ReplicationControllers",
+        "ResourceQuotas",
+        "RoleBindings",
+        "Roles",
+        "ServiceAccounts",
+        "Services",
+        "StatefulSets"
+    ],
+    Filters: {
+    LabelSelector: "",
+    Namespaces: ".*"
+    },
+    Server: {
+        advertiseaddress: "sonobuoy-master:8080",
+        bindaddress: "0.0.0.0",
+        bindport: 8080,
+        timeoutseconds: 3600
+    },
+    PluginNamespace: "heptio-sonobuoy",
+    Plugins: [
+          {name: "systemd_logs"},
+          {name: "e2e"}
+    ]
+};
+
+local plugins = {
+  "systemdlogs.yaml": 'name: systemd_logs
```

Comment:
Hmm, it seems to me you could write this as a normal JSON object and then call `std.toString(systemdLogs)` on it to get a string? Am I missing something crucial?